### ISSUE TITLE
cascade=delete does cause a DBAL exception "no such column"

### ISSUE THREAD

**Issue by Vassyli** [lotgd/core#85]
During the changes for the modules and preparation of release 0.2, I've found that doctrine throws a DBAL Exception if a scene gets deleted which has connection groups associated.

```1) ModuleTest::testUnregister
Doctrine\DBAL\DBALException: An exception occurred while executing 'DELETE FROM scene_connection_groups WHERE scene = ? AND name = ?' with params [null, "lotgd\/module-village\/outside"]:

SQLSTATE[HY000]: General error: 1 no such column: scene
```

**Comment by austenmc** [comment]
Loving our tests btw :)

**Comment by Vassyli** [comment]
Fixes in #84
